### DISCUSSION TITLE
Utility to compare two gdu exports for folder size changes

### DISCUSSION THREAD

**Discussion by noelhibbard** [dundee/gdu#372] [General]
I am trying to track down where storage is getting used and figured there would be a utility to compare two gdu exports and visualize the size changes between the two. Does anyone know of a utility that does this? I started writing my own utility to do this but find it hard to believe there isn't one out there already. I don't even see anyone discussing a feature or utility like this.

**Comment by dundee** [comment]
If you want so see a graph tracking the usage over time, you can use Prometheus with https://github.com/dundee/disk_usage_exporter
